ci/build_cpp.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -e
+
+# Check environment
+source ci/check_environment.sh
+
+# Update env vars
+source rapids-env-update
```

Comment:
Let's move `rapids-env-update` above `ci/check_environment` since `rapids-env-update` may add some environment variables that we want to view.